packages/nextjs/app/builders/0x2b2542b313385AF7453623765383CE3cc5D3c9cF/page.tsx

```diff
@@ -0,0 +1,41 @@
+import Image from "next/image";
+import type { NextPage } from "next";
+
+const PersonalPage: NextPage = () => {
+  return (
+    <>
+      <div className="flex-grow bg-base-300 w-full mt-16 px-8 py-12">
+        <h1 className="text-center">
+          <span className="block text-2xl mb-2">My </span>
+          <span className="block text-4xl font-bold">Profile</span>
+        </h1>
+        <div className="card card-side bg-base-100 shadow-xl">
+          <figure>
+            <Image src="/assets/profileimage.jpeg" alt="Temitope O Hassan" width={399} height={399} />
```

Comment:
![Screenshot 2024-05-14 at 4 16 58 PM](https://github.com/BuidlGuidl/batch5.buidlguidl.com/assets/80153681/a2e9f98d-8454-47b3-b5ee-4c7f21c3b850)


I think you forgot to update the correct path, it should be `assets/temitopeohassan.jpeg`  ?

Reply:
I have corrected the error on the page